src/mkdocs_gen_nav_plugin/plugin.py

```diff
@@ -1,7 +1,8 @@
 import os
-from mkdocs.plugins import BasePlugin
+from mkdocs.plugins import BasePlugin, get_plugin_logger
 from mkdocs.config import config_options
 
+log = get_plugin_logger(__name__)
```

Comment:
get_plugin_logger is supported from 1.5+
I would like to support 1.4.2+
So, we can use logger like this for now?
```
log = logging.getLogger('mkdocs.plugins.' + __name__)
```

Reply:
I will change this, this week